setup.py

```diff
@@ -0,0 +1,31 @@
+import pathlib
+from setuptools import setup, find_packages
+
+HERE = pathlib.Path(__file__).parent
+
+README = (HERE / "README.md").read_text()
+install_requires = [line.rstrip() for line in (HERE / "requirements.txt").read_text().splitlines()]
+
+setup(
+    name="hca-to-scea",
+    version="1.0.0",
```

Comment:
I think this should be v0.1.0. See [https://semver.org/](Semver versioning). New projects start at v0